src/AlgebraicGeometry/Schemes/CoveredSchemes/Objects/Properties.jl

```diff
@@ -53,7 +53,8 @@ function _jacobian_criterion(X::CoveredScheme{<:Field})
 
   dec_info = decomposition_info(default_covering(X))
   for (V, fs) in dec_info
-    R = base_ring(OO(V))
+    R_quotient = OO(V)
+    R_quotient isa MPolyQuoRing ? R = base_ring(R_quotient) : R = R_quotient
```

Comment:
Are you 100% sure that you never see an MPolyLocalizedRing or an MPolyQuoLocalizedRing here? This is the general case of covered schemes here, not the projective scheme case. We could see for instance a complement of a hypersurface as a chart (which gives the right answer for base_ring, by the way). 

I would do this fix the other way around: 

R_quotient isa MPolyRing ? R = R_quotient : R = base_ring(R_quotient)

Reply:
I am not entirely sure how the algorithms would work for localized rings, in particular if the localization results in a nonfinitely generated ring. So I left it as it is for now.